src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Calling `Color.values()` inside a loop or a frequently called method can be inefficient because it creates a new array every time it is called. Consider storing the result of `Color.values()` in a class-level constant array and reusing it.